src/api/supersetGateway.js

```diff
@@ -0,0 +1,57 @@
+import { useConfig } from '@dhis2/app-service-config'
+import { useCallback, useMemo } from 'react'
+
+const useSupersetApiUrl = (resource) => {
```

Comment:
Perhaps each of these hooks could use a comment block with a little explanation?